Dockerfile

```diff
@@ -10,13 +10,13 @@ ENV POETRY_NO_INTERACTION=1 \
 
 RUN pip install poetry==${POETRY_VERSION}
 
-COPY pyproject.toml poetry.lock ./
+COPY pyproject.toml poetry.lock README.md ./
```

Comment:
Does the process need the readme file?

Reply:
yes. It needs a README.md file to for `poetry install` to works now